cmd/snap/cmd_debug_bootvars.go

```diff
@@ -23,20 +23,27 @@ import (
 	"github.com/jessevdk/go-flags"
 
 	"github.com/snapcore/snapd/boot"
+	"github.com/snapcore/snapd/i18n"
 )
 
-type cmdBootvars struct{}
+type cmdBootvars struct {
+	UC20 bool   `long:"uc20"`
+	Dir  string `long:"dir"`
+}
 
 func init() {
 	cmd := addDebugCommand("boot-vars",
 		"(internal) obtain the snapd boot variables",
 		"(internal) obtain the snapd boot variables",
 		func() flags.Commander {
 			return &cmdBootvars{}
-		}, nil, nil)
+		}, map[string]string{
+			"uc20": i18n.G("Whether to use uc20 boot vars or not"),
+			"dir":  i18n.G("What directory to look for boot variables in"),
+		}, nil)
 	cmd.hidden = true
```

Comment:
Opened https://github.com/snapcore/snapd/pull/8491 to not hide the command on core.